src/main/webapp/app/modules/canvas/filter/filter-panel.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import { ActionButton, Flex, View, Text, Button, Divider } from '@adobe/react-spectrum';
+import $ from 'jquery';
+import { IRootState } from 'app/shared/reducers';
+import './filter-panel.scss';
+import Minimize from '@spectrum-icons/workflow/Minimize';
+import Search from '@spectrum-icons/workflow/Search';
+import Maximize from '@spectrum-icons/workflow/Maximize';
+import FilterElement, { IFilterState } from 'app/modules/canvas/filter/filter-element';
+import uuid from 'react-uuid';
+import { getDatasourcesFeaturesEntities as getfeatureEntities } from 'app/entities/feature/feature.reducer';
+import { getVisualizationData, ValidateFields } from '../visualization/util/visualization-render-utils';
+import { FilterParameterService } from './filter-parameters-service';
+import { updateSelectedFilter } from 'app/entities/visualmetadata/visualmetadata.reducer';
+
+export interface IFilterPanelProp extends StateProps, DispatchProps {}
+
+const FilterPanel = (props: IFilterPanelProp) => {
+  const [isFilterMinimize, setFilterMinimize] = useState(true);
+  const [isFilterPanelClose, setFilterPanelClose] = useState(props.isFilterOpen);
+
+  const renderVisualizationById = item => {
+    if (ValidateFields(item.fields)) {
+      getVisualizationData(item, props.view);
+    } else {
+      $(`.loader-${item.id}`).hide();
```

Comment:
hide show should handle using flag coming from redux reducer. do not use jquery stuff